src/CMakeLists.txt

```diff
@@ -428,9 +431,11 @@ target_link_libraries(${EXECUTABLE_CPPREGIONTEST} ${COMMON_LIBS})
 set_target_properties(${EXECUTABLE_CPPREGIONTEST} PROPERTIES COMPILE_FLAGS ${COMMON_COMPILE_FLAGS})
 set_target_properties(${EXECUTABLE_CPPREGIONTEST}
                       PROPERTIES LINK_FLAGS "${COMMON_LINK_FLAGS}")
+if(UNIX) # or UNIX like (i.e. APPLE and CYGWIN)
```

Comment:
This, and the other if(UNIX), are removing the Utility projects tests_cpp_region and tests_unit. Although they generate projects, those projects fail in AppVeyor and VS. Hence the un-wrap of the solution in the AppVeyor yaml. 
The unit and cpp_region tests work fine (although have bugs unrelated from Cap'n'Proto). I'll undo this change. It'll appear elsewhere for 0.3.0